packages/pytest-simcore/src/pytest_simcore/rabbit_service.py

```diff
@@ -141,12 +140,11 @@ async def ensure_parametrized_queue_is_empty(
     rabbitmq_client = create_rabbitmq_client("pytest-purger")
 
     async def _queue_messages_purger() -> None:
-        with suppress(aio_pika.exceptions.ChannelClosed):

```

Comment:
Q: how was this related to redis and the tasks?

Reply:
this is not, but that is useless and generating issues